src/main/java/org/spongepowered/api/data/type/BodyPart.java

```diff
@@ -25,9 +25,10 @@
 package org.spongepowered.api.data.type;
 
 import org.spongepowered.api.CatalogType;
+import org.spongepowered.api.text.translation.Translatable;
 import org.spongepowered.api.util.annotation.CatalogedBy;
 
 @CatalogedBy(BodyParts.class)
-public interface BodyPart extends CatalogType {
+public interface BodyPart extends CatalogType, Translatable {
```

Comment:
Well, now I'm not really sure what this PR does save for a whole bunch of assorted changes all around the codebase. Once again, a lot of these changes are out of scope. Please split it up into smaller, more self-contained PRs.